### PR TITLE
feat: org issues alert for org admins

### PR DESCRIPTION
**Pull request by BlankParticle**
## What does this PR do?

Added a new endpoint and a alerts to show admins about issues with their orgs.
Currently shows misconfigured domains only.

Edit:
Turns out I had not accounted for the fact that the DNS check uppdated Postal's db only. So I had to extract out the DNS check logic and make 2 endpoints, one for trpc and one for worker to trigger.

Worker now calls to Platform for DNS check and Platform calls mail-bridge and figures out if dns is valid or not, sends realtime events if issues are found.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by BlankParticle** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/un/inbox/547?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#547** <a href="https://app.graphite.dev/github/pr/un/inbox/547?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#545** <a href="https://app.graphite.dev/github/pr/un/inbox/545?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @BlankParticle and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortCode]/_components/org-issues-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          <div className="flex flex-col gap-2 whitespace-pre">
+            {issues.map((issue, i) => (
+              <div key={i}>{issue}</div>
+            ))}
+          </div>
+        </DialogHeader>
+      </DialogContent>
+    </Dialog>
+  );
+}
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortCode]/_components/org-issues-dialog.tsx:46]: issue mentions banner, not dialog

File: apps/web/src/app/[orgShortCode]/layout.tsx [hunk 2]
```diff
@@ -85,6 +88,12 @@ export default function Layout({
           {hasEmailIdentity && !hasEmailIdentity.hasIdentity && (
             <ClaimEmailIdentity />
           )}
+          {orgIssues && orgIssues.issues.length > 0 && (
+            <OrgIssuesDialog
+              orgs={orgIssues.orgs}
+              issues={orgIssues.issues}
+            />
+          )}
```
Comment (McPizza0) [inline apps/web/src/app/[orgShortCode]/layout.tsx:96]: this approach seems really hacky
dosnt flow right

**Review by McPizza0 — COMMENTED** [review]
@BlankParticle i dont see how we will trigger the alert via realtime system? 👀

**Comment by BlankParticle** [comment]
> @BlankParticle i dont see how we will trigger the alert via realtime system? 👀
I had to change some of the DNS check logic to get it working. check the Edited part of the PR description for more details

File: apps/platform/app.ts [hunk 3]
```diff
@@ -48,6 +49,10 @@ app.get('/health', (c) => {
 app.route('/auth', authApi);
 app.route('/realtime', realtimeApi);
 
+// Service Endpoints
```
Comment (McPizza0) [inline apps/platform/app.ts:52]: // Services Endpoing for intra application/services procedures/api

**Review by McPizza0 — APPROVED** [review]

**Pull request merged**